Trail_Project/Strategy_test2.py

```diff
@@ -38,6 +38,16 @@ def calculate_risk(self, patient):
             return "Low risk."
         return "Moderate risk due to age."
 
+#Added by Chetan Chhetri ___ the calculation for heart attack risk   
```

Comment:
Please co-ordinate with Shiva Bathini to add futher health diseases calculations in effective methods